libmscore/edit.cpp

```diff
@@ -1435,6 +1435,10 @@ void Score::cmdAddOttava(OttavaType type)
                        cr2 = cr1;
                   Ottava* ottava = new Ottava(this);
                   ottava->setOttavaType(type);
+                  if (type == OttavaType::OTTAVA_8VB/* || type == OttavaType::OTTAVA_15MB || type == OttavaType::OTTAVA_22MB*/) {
```

Comment:
No need to check for `OttavaType::OTTAVA_15MB` and `OttavaType::OTTAVA_22MB`, as there's no shortcut for those